auth/Makefile

```diff
@@ -0,0 +1,90 @@
+DIR=certs
+PROTOCOL=x509
+KEY_SIZE=4096
+VALIDITY_DAYS=9999
+
+CA_KEY=$(DIR)/ca-key.pem
+CA_CRT=$(DIR)/ca-crt.pem
+
+VMD_SERVER_KEY=$(DIR)/vmd-server-key.pem
+VMD_SERVER_CSR=$(DIR)/vmd-server-csr.pem
+VMD_SERVER_CRT=$(DIR)/vmd-server-crt.pem
+
+VMD_CLIENT_KEY=$(DIR)/vmd-client-key.pem
+VMD_CLIENT_CSR=$(DIR)/vmd-client-csr.pem
+VMD_CLIENT_CRT=$(DIR)/vmd-client-crt.pem
+
+COUNTRY=FI
+STATE=Uusimaa
+LOCALITY=Helsinki
+ORGANIZATION=VMD
+ORGANIZATION_UNIT=VMD
+
+CA_COMMON_NAME=ca-vmd
+VMD_SERVER_COMMON_NAME=vmd-server
+VMD_CLIENT_COMMON_NAME=vmd-client
+
+PASSWORD=1234
+CA_SUBJECT=/C=$(COUNTRY)/ST=$(STATE)/L=$(LOCALITY)/O=$(ORGANIZATION)/OU=$(ORGANIZATION_UNIT)/CN=$(CA_COMMON_NAME)
+SERVER_SUBJECT=/C=$(COUNTRY)/ST=$(STATE)/L=$(LOCALITY)/O=$(ORGANIZATION)/OU=$(ORGANIZATION_UNIT)/CN=$(VMD_SERVER_COMMON_NAME)
+CLIENT_SUBJECT=/C=$(COUNTRY)/ST=$(STATE)/L=$(LOCALITY)/O=$(ORGANIZATION)/OU=$(ORGANIZATION_UNIT)/CN=$(VMD_CLIENT_COMMON_NAME)
+
+all: ca server client
+
+ca: $(DIR)
+	@echo "Generating CA key and certificate"
+	@openssl req -new -$(PROTOCOL)	\
+		-days $(VALIDITY_DAYS)		\
+		-keyout $(CA_KEY)			\
+		-out $(CA_CRT)				\
+		-passout pass:$(PASSWORD)	\
+		-subj "$(CA_SUBJECT)"
+
+server: $(CA_CRT) 
+	@echo "Generating vmd-server key and certificate signing request"
+	@openssl genrsa -out $(VMD_SERVER_KEY) $(KEY_SIZE)
+	@openssl req -new				\
+		-key $(VMD_SERVER_KEY)		\
+		-out $(VMD_SERVER_CSR)		\
+		-passin pass:$(PASSWORD)	\
+		-subj "$(SERVER_SUBJECT)"
+	@openssl $(PROTOCOL) -req				\
+		-days $(VALIDITY_DAYS)				\
+		-in $(VMD_SERVER_CSR)				\
+		-CA $(CA_CRT)						\
+		-CAkey $(CA_KEY) -CAcreateserial	\
+		-passin pass:$(PASSWORD)			\
+		-out $(VMD_SERVER_CRT)
+	@openssl verify -CAfile $(CA_CRT) $(VMD_SERVER_CRT)
+
+client: $(CA_CRT)
+	@echo "Generating client key and certificate signing request"
```

Comment:
The same here - `Generating vmd-client sample/testing key and certificate signing request`